app/v4/retrievePeriodSummary/RetrievePeriodSummaryConnector.scala

```diff
@@ -47,7 +47,7 @@ class RetrievePeriodSummaryConnector @Inject() (val http: HttpClient, val appCon
 
     request match {
       case def2: Def2_RetrievePeriodSummaryRequestData =>
-        val downstreamUri = TaxYearSpecificIfsUri[Def2_RetrievePeriodSummaryResponse](
+        val downstreamUri = IfsUri[Def2_RetrievePeriodSummaryResponse](
           s"income-tax/${def2.taxYear.asTysDownstream}/$nino/self-employments/$businessId/periodic-summary-detail?from=$fromDate&to=$toDate")
         val result = get(downstreamUri)
         result
```

Comment:
Remove this feature switch `featureSwitches.isDesIf_MigrationEnabled` with `featureSwitches: SeBusinessFeatureSwitches` thanks. It is already enabled in [production](https://github.com/hmrc/app-config-production/blob/main/self-employment-business-api.yaml#L67) so should be removed as part of these changes. Also, remove `isDesIf_MigrationEnabled` entirely, thanks.